src/components/atoms/ModalCreateTask.jsx

```diff
@@ -0,0 +1,154 @@
+import { useMutation } from "@apollo/client";
+import React, { useState } from "react";
+import Modal from "react-modal";
+import styled from "styled-components";
+import createTask from "../../graphgl/mutations/create_tasks";
```

Comment:
better use camelCase for naming components, like a best practice
```suggestion
import createTask from "../../graphgl/mutations/createTasks";
```

Reply:
ok, i changed all components for this style